p2panda-rs/src/document/document_view_id.rs

```diff
@@ -59,6 +59,22 @@ impl DocumentViewId {
         graph_tips.sort();
         graph_tips
     }
+
+    /// Can be used as a human-readable representation of a document view id.
+    ///
+    /// The return value contains all view graph tips' last 6 characters
+    /// concatenated with an underscore.
+    pub fn to_short_string(&self) -> String {
```

Comment:
Not sure if it will ever be used outside of `Human`, should we make this only visible on a crate level?

Reply:
Yes, good catch!